src/composables/accounts.ts

```diff
@@ -11,14 +11,17 @@ export function useAccounts({ store }: IDefaultComposableOptions) {
   /**
    * Accounts data formatted as the form select options
    */
-  const accountsSelectOptions = computed(
-    (): IFormSelectOption[] => accounts.value.map((acc) => ({
-      text: getAccountNameToDisplay(acc),
+  function prepareAccountsSelectOptions(withName = false): IFormSelectOption[] {
+    return accounts.value.map((acc) => ({
+      text: withName ? getAccountNameToDisplay(acc) : acc.address,
       value: acc.address,
       address: acc.address,
-    })),
-  );
+      listOnlyText: getAccountNameToDisplay(acc),
+    }));
+  }
+  const accountsSelectOptions = computed(() => prepareAccountsSelectOptions(true));
 
+  const plainAccountSelectOptions = computed(() => prepareAccountsSelectOptions());
```

Comment:
I suggest to rename this to `accountSelectOptionsPlain` to follow the same naming as the `accountSelectOptions`. Also I suggest to add empty line after this one.